driver/config/provider_test.go

```diff
@@ -279,7 +279,7 @@ func TestViperProviderValidates(t *testing.T) {
 	// ttl
 	assert.Equal(t, 2*time.Hour, c.ConsentRequestMaxAge())
 	assert.Equal(t, 2*time.Hour, c.AccessTokenLifespan())
-	assert.Equal(t, 2*time.Hour, c.RefreshTokenLifespan())
+	assert.Equal(t, -1*time.Nanosecond, c.RefreshTokenLifespan())
```

Comment:
Could you please add this as a separate test, rather than change the behavior of current tests? Thank you!

Reply:
Sorry for the late response!
I moved that to a separate test.  :)